.github/workflows/kind.yml

```diff
@@ -50,8 +50,10 @@ jobs:
 
     - name: Verify format
       run: |
+        make fmt-dependencies
```

Comment:
This was a mistake in my original PR to the propagator where I had forgotten that `make fmt` actually also calls the `fmt-dependencies` target. Could you please remove this line?